setup.py

```diff
@@ -29,7 +31,6 @@
     zip_safe=False,
     install_requires=[
         'requests',
-        'sphinx>=1.2.0',
```

Comment:
Any reason not to make this a `docs` extra?

https://pythonhosted.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies